src/client/middlewares/peerRTC.js

```diff
@@ -1,10 +1,10 @@
-import { params } from '../constants';
 import { RTCConnection } from '../actions/connexion';
 
 export default peer => ({ dispatch, getState }) => {
   peer.on('open', (id) => {
     console.log(`My peer ID is: ${id}`, getState());
-    getState().socket.emit('peer', { id, room: params.server.url });
+    const { socket } = getState();
+    socket.emit('/game', { path: '/join', id, room: window.location.hash.slice(1), socket: socket.id });
```

Comment:
no need to add socket.id in param object because you can have the socket id simply on the backend in socket object

Reply:
sokcet id client-side is different from the one on the back side, + you'll have to keep those client sockets id in order to emit to a particular person etc etc

Reply:
"sokcet id client-side is different from the one on the back side" ??